ra/ra.go

```diff
@@ -566,108 +566,115 @@ func (ra *RegistrationAuthorityImpl) checkNewOrdersPerAccountLimit(ctx context.C
 
 // NewAuthorization constructs a new Authz from a request. Values (domains) in
 // request.Identifier will be lowercased before storage.
-func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, request core.Authorization, regID int64) (core.Authorization, error) {
-	identifier := request.Identifier
-	identifier.Value = strings.ToLower(identifier.Value)
+func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, request *rapb.NewAuthorizationRequest) (*corepb.Authorization, error) {
```

Comment:
```suggestion
func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, req *rapb.NewAuthorizationRequest) (*corepb.Authorization, error) {
```
Nit; with the exception of `NewRegistration` the other (five) methods that accept a request object use the variable `req` instead of request.

Reply:
Fair enough. Done!